blueprints/app/files/.github/workflows/ci.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
+
+jobs:
+
+  lint:
+    name: "Lint"
+    env:
+      CI: true
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install node
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+
+    - name: Get <%= yarn ? 'yarn' : 'npm' %> cache directory path
+      id: <%= yarn ? 'yarn' : 'npm' %>-cache-dir-path
+      run: echo "::set-output name=dir::$(<%= yarn ? 'yarn config get cacheFolder' : 'npm config get cache' %>)"
+    - uses: actions/cache@v2
+      id: <%= yarn ? 'yarn' : 'npm' %>-cache
+      with:
+        path: ${{ steps.<%= yarn ? 'yarn' : 'npm' %>-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-modules-${{ hashFiles('<%= yarn ? '**/yarn.lock' : '**/package-lock.json' %>') }}
+    - name: Install Dependencies
+      run: <%= yarn ? 'yarn' : 'npm' %> install
+    - name: Lint
+      run: <%= yarn ? 'yarn' : 'npm run' %> lint
+
+  floating:
```

Comment:
In an app, this wouldn't be floating dependencies, maybe it should be named `Test`? 
 `npm install` and `yarn install` will use the lockfiles.